// const httpError = (res, err) => {
//   console.error(err);
//   res.status(500);
//   res.send({
//     error:
//       "server encountered an unexpected condition that prevents it from completing the request",
//   });
// };

// module.exports = { httpError };
